Labelled the axes and titles in all the graphs.

paras009 opened the pull request:
@hugobowne  Please reveiw.

hugobowne commented:
thanks for this, @paras009. 

Some details:

- in some figure titles, "patients" is misspelt.
- x-axis label 'Year' should be 'Date' <-- my original mistake, IIRC
- edit 'Time Series Analysis" to "Time Series" in titles , as it's not really an analysis yet, it's plotting.

Could you make these minor edits?

paras009 commented:
Updated the changes.
Additionally, the dataset names in the main data repository were changed so I changed the urls in the notebook.
@hugobowne please review.

hugobowne commented:
My understanding is that all these edits are reflected in your other PR, @paras009 https://github.com/hugobowne/COVID-19-EDA-tutorial/pull/3

Feel free to re-open if I'm incorrect.
